pkg/apis/smith/v1/types.go

```diff
@@ -187,6 +187,9 @@ func (bs *BundleStatus) ShortString() string {
 // ResourceName is a reference to another Resource in the same bundle.
 type ResourceName string
 
+// PluginName is a name of a plugin to be invoked.
```

Comment:
what's the advantage of this? (curiosity)

Reply:
Type safety